modules/lowvram.py

```diff
@@ -135,12 +268,22 @@ def first_stage_model_decode_wrap(z):
         diff_model.input_blocks, diff_model.middle_block, diff_model.output_blocks, diff_model.time_embed = stored
 
         # install hooks for bits of third model
-        diff_model.time_embed.register_forward_pre_hook(send_me_to_gpu)
-        for block in diff_model.input_blocks:
-            block.register_forward_pre_hook(send_me_to_gpu)
-        diff_model.middle_block.register_forward_pre_hook(send_me_to_gpu)
-        for block in diff_model.output_blocks:
-            block.register_forward_pre_hook(send_me_to_gpu)
+
+        if use_streamlined_lowvram:
+            # put it into pinned memory to achieve data transfer overlap
+            diff_model.time_embed._apply(lambda x: x.pin_memory())

```

Comment:
Specifying `device` parameter will let `pin_memory` offload to other non-CUDA backends (e.g. IPEX)
```suggestion
            diff_model.time_embed._apply(lambda x: x.pin_memory(device=devices.get_optimal_device_name()))
```